heltour/tournament/models.py

```diff
@@ -216,6 +216,7 @@ def extract(sp):
             if reg is not None:
                 info.update({
                     'date_created': reg.date_created.isoformat(),
+                    'peak_classical_rating': reg.peak_classical_rating or 0,
```

Comment:
What is this for?  Why 0?

Reply:
It's pointless. It's copy-pasta from where I looked up the peak_classical_rating field name. I've removed it.